blog/2024-02-01-py-client-v4-release/index.mdx

```diff
@@ -0,0 +1,345 @@
+        />
+        <p><small>The search query returns a parsed response, with the <code>.objects</code> property containing the retrieved Weaviate objects.</small></p>
+    </TabItem>
+    <TabItem value="rag" label="NearText + RAG">
+        <FilteredTextBlock
+            text={PythonCode}
+            startMarker="# START NearTextGenerateExample"
+            endMarker="# END NearTextGenerateExample"
+            language="py"
+        />
+        <p><small>The RAG query is very similar to the equivalent search query, but using methods in the <code>generate</code> submodule.</small></p>
+    </TabItem>
+    <TabItem value="filter" label="Aggregate + Filter">
+        <FilteredTextBlock
+            text={PythonCode}
+            startMarker="# START AggregateCountExample"
+            endMarker="# END AggregateCountExample"
+            language="py"
+        />
+        <p><small>This example retrieves a count of objects that match the filter criteria.</small></p><br/>
+    </TabItem>
+</Tabs>
+
+#### Responses
+
+As mentioned before, the default behavior now is to return all properties (except `blob`s) by default, as well as object uuids.
+
+For less-often used attributes such as cross-references, metadata (previously under `_additional`) and vectors must be requested specifically. This is done through the `return_references`, `return_metadata` and `include_vector` parameters.
+
+Responses from Weaviate are now parsed into instances of custom classes. For example, each query response will have a list of objects in the `objects` attribute, and each object will have a set of attributes itself. Responses for RAG queries will include a `generated` attribute at the top level for `grouped_task` query, and each object will also have a `generated` attribute for `single_prompt` queries.
+
+### Iterator
+
+The client includes an iterator to make convenient use of the `cursor` API. This is useful for retrieving large numbers of objects, such as when manually exporting or migrating a collection.
+
+<Tabs groupId="languages">
+    <TabItem value="basic" label="Default">
+        <FilteredTextBlock
+            text={PythonCode}
+            startMarker="# IteratorBasic"
+            endMarker="# END IteratorBasic"
+            language="py"
+        />
+    </TabItem>
+    <TabItem value="oneprop" label="Selected properties">
+        <FilteredTextBlock
+            text={PythonCode}
+            startMarker="# IteratorAnswerOnly"
+            endMarker="# END IteratorAnswerOnly"
+            language="py"
+        />
+    </TabItem>
+    <TabItem value="meta" label="Metadata Only">
+        <FilteredTextBlock
+            text={PythonCode}
+            startMarker="# IteratorMetadataOnly"
+            endMarker="# END IteratorMetadataOnly"
+            language="py"
+        />
+    </TabItem>
+</Tabs>
+
+## How to get started
+
+:::caution TODO
+- Installation
+- Where to find useful docs
+    - V4 client page
+    - Code snippets (howtos etc.)
```

Comment:
I'd add some cross reference examples to show much easier it is to create and request them

Reply:
Done